cypress.json

```
@@ -1,5 +1,5 @@
 {
-  "baseUrl": "https://localhost:8081/index.php/",
+  "baseUrl": "http://localhost:8081/index.php/",
   "viewportWidth": 1280,
   "viewportHeight": 720,
   "experimentalSessionAndOrigin": true
```

```suggestion
  "experimentalSessionAndOrigin": true,
  "projectId": "b6myzg"
```

Created a project and added the secret to the repo.

---

https://cloud.cypress.io/projects/b6myzg/runs

---

Thanks @juliushaertl - added.